CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## v4.2.3
```

Comment:
as you're not patching an issue but adding functionality it should be a minor update 
```suggestion
## v4.3.0
```

same in pyproject.toml too

Reply:
Thanks Matt, I've made the update!